src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -3340,7 +3331,7 @@ namespace ZoneTempPredictorCorrector {
 				LoadToHeatingSetPoint = AIRRAT( ZoneNum ) * ( TempZoneThermostatSetPoint( ZoneNum ) - ZoneT1( ZoneNum ) ) + TempDepZnLd( ZoneNum ) * ( TempZoneThermostatSetPoint( ZoneNum ) ) - TempIndZnLd( ZoneNum );
 			}
 			if ( RAFNFrac > 0.0 ) LoadToHeatingSetPoint = LoadToHeatingSetPoint / RAFNFrac;
-			if ( TempControlledZone( RelativeZoneNum ).HeatOffFlag ) {
+			if ( HeatOffFlag ) {
 				LoadToHeatingSetPoint = 0.0;
```

Comment:
@lgu1234 I don't understand why `LoadToHeatingSetPoint` is set to 0.0 here. I think I follow the `HeatOffFlag` logic, but once the zone reaches the setpoint+cutoff and `HeatOffFlag=true`, then the thermostat setpoint should revert to the lower temperature and the LoadToHeatSetPoint should be based on that (the original setpoint).
@Myoldmopar This behaves far better than what's in develop, but I think there's one more improvement that could be made. I can experiment with that tomorrow or we merge this as-is and add the improvement post-release if it proves to be useful.

Reply:
I will move this to the end of my list of tabs as they are ordered by when they should drop in.  If you have some time to experiment tomorrow that would be fine.  If it doesn't make it into this, that's fine too.

Reply:
@mjwitte Thanks for your comments. I think of the same way by removing line LoadToHeatingSetPoint = 0.0. I will test tonight and let you know what happens.